spec/message.abnf

```diff
@@ -41,9 +41,7 @@ variable       = "$" name
 
 literal          = quoted-literal / unquoted-literal
 quoted-literal   = "|" *(quoted-char / escaped-char) "|"
-unquoted-literal = name / number-literal
-; number-literal matches JSON number (https://www.rfc-editor.org/rfc/rfc8259#section-6)
-number-literal   = ["-"] (%x30 / (%x31-39 *DIGIT)) ["." 1*DIGIT] [%i"e" ["-" / "+"] 1*DIGIT]
+unquoted-literal = 1*name-char
```

Comment:
This change loses the optional `bidi` productions around unquoted (because they are in `name`). This could be fixed and is probably cleaner if we add the `o` (optional whitespace) production to the end of `attribute` and `option` (all of the other functions already have optional/required whitespace after a literal).

Reply:
Agreed.

Reply:
That should not be necessary, as whitespace is always allowed after `option` and `attribute`.